client/src/pages/ChefSearch.js

```diff
@@ -35,6 +35,7 @@ function ChefSearch() {
     const location = [city, region, country].join(", ");
     const [radiusKm, setRadiusKm] = useState(100);
     const [chefs, setChefs] = useState([]);
+    console.log(chefs);
```

Comment:
```suggestion
```

Reply:
man. I really tried to catch all those this time.

Reply:
:smile: I have a checklist I personally run on all my code before pushing to a PR, and it includes doing a find all for console.log. I always catch some